docs/reference/feel/language-guide/feel-variables.md

```diff
@@ -1,23 +1,24 @@
 ---
 id: feel-variables 
 title: Variables
+description: "This document outlines variables and examples."
 ---
 
-### Access Variables
+### Access variables
 
 Access the value of a variable by its variable name.
 
 ```js
 a + b
 ```
 
-If the value of the variable is a context then a [context entry can be accessed](/docs/reference/feel/language-guide/feel-context-expressions#get-entry--path) by its key. 
+If the value of the variable is a context, a [context entry can be accessed](/docs/reference/feel/language-guide/feel-context-expressions#get-entry--path) by its key.
 
 ```js
 a.b
 ```
 
-:::tip Tip
+:::note
```

Comment:
:thought_balloon: Same here.

![image](https://user-images.githubusercontent.com/4305769/155109265-178fba5f-2e41-42d1-8b10-7b8a48f903b0.png)

vs.

![image](https://user-images.githubusercontent.com/4305769/155109317-343f14bd-1b5a-4aed-a739-3edfd8abcd79.png)

Reply:
Adjusted in latest commit.